app/views/shelters/new.html.erb

```diff
@@ -8,21 +8,12 @@
 <div class="container">
   <div class="row">
     <div class="col-12 left">
-      <form action="/shelters" method="post">
-        <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
-        <label for="name">Name:</label>
-        <input type="text" id="name" name="shelter[name]"><br><br>
-        <label for="address">Address:</label>
-        <input type="text" id="address" name="shelter[address]"><br><br>
-        <label for="city">City:</label>
-        <input type="text" id="city" name="shelter[city]"><br><br>
-        <label for="state">State:</label>
-        <input type="text" id="state" name="shelter[state]"><br><br>
-        <label for="zip">Zip:</label>
-        <input type="text" id="zip" name="shelter[zip]"><br><br>
-        <input type="submit" value="Create Shelter">
+      <%= render partial: 'form', locals: {
+                                      path: "/shelters",
+                                      method: :post,
```

Comment:
looks good!! I imagine partial will save us a lot of time from now on, using them from the start.